diesel/src/pg/expression/expression_methods.rs

```diff
@@ -370,6 +370,50 @@ pub trait PgArrayExpressionMethods: Expression + Sized {
     {
         ArrayIndex::new(self, other.as_expression())
     }
+
+    /// Concatenates two PostgreSQL Arrays using the `||` operator.
```

Comment:
Please follow the same pattern of documentation from other methods. `contains` is a good example.